wowup-electron/src/assets/i18n/de.json

```diff
@@ -193,43 +193,43 @@
       "MULTIPLE_PROVIDERS_TOOLTIP": "Dieses Addon hat mehrere Anbieter",
       "PAGE_CONTEXT_FOOTER": {
         "ADDONS_INSTALLED": "{count} {count, plural, =1{Addon} other{Addons}}",
-        "JOIN_DISCORD": "Schreibe mit uns auf Discord",
+        "JOIN_DISCORD": "Rede mit uns auf Discord",
         "PATREON_SUPPORT": "Unterstütze WowUp auf Patreon",
         "SEARCH_RESULTS": "{count} {count, plural, =1{Treffer} other{Treffer}}",
         "VIEW_GITHUB": "Schau dir den Code auf GitHub an",
         "VIEW_GUIDE": "Schaue dir unseren Leitfaden an, um zu sehen, was WowUp kann"
       },
-      "RESCAN_FOLDERS_BUTTON": "Ordner erneut scannen",
-      "RESCAN_FOLDERS_BUTTON_TOOLTIP": "Scannen Sie Ihren Client-Ordner nach installierten Addons",
-      "RESCAN_FOLDERS_CONFIRMATION_DESCRIPTION": "Wenn du ein erneutes scannen durchführst, können deine Addon Informationen verloren gehen. Dieser Vorgang braucht einen Moment.",
-      "RESCAN_FOLDERS_CONFIRMATION_TITLE": "Neu scannen starten?",
+      "RESCAN_FOLDERS_BUTTON": "Deine Ordner erneut scannen",
+      "RESCAN_FOLDERS_BUTTON_TOOLTIP": "Scannt deinen Clientordner nach bereits installierten Addons",
+      "RESCAN_FOLDERS_CONFIRMATION_DESCRIPTION": "Ein erneuter Scan wird versuchen, bereits installierte Addons zu identifizieren. Dies kann bereits bekannte Addon-Informationen zurücksetzen. Mache dies nur falls bestimmte Addons nicht erkannt werden oder Addon-Versionen nicht richtig angezeigt werden. Dieser Scan wird nie deine installierten Addons löschen, sondern nur das was in WowUp angezeigt wird.\n\nDies kann einen Moment dauern.",
+      "RESCAN_FOLDERS_CONFIRMATION_TITLE": "Erneut scannen?",
       "SPINNER": {
-        "GATHERING_ADDONS": "Sammle Addons...",
-        "UPDATING": "Aktualisiere {updateCount}/{addonCount}",
-        "UPDATING_WITH_ADDON_NAME": "Aktualisiere {updateCount}/{addonCount}\n{clientType}: {addonName}"
+        "GATHERING_ADDONS": "Addons sammeln...",
+        "UPDATING": "{updateCount}/{addonCount} aktualisiert",
+        "UPDATING_WITH_ADDON_NAME": "{updateCount}/{addonCount} aktualisiert\n{clientType}: {addonName}"
       },
       "TABLE": {
         "ADDON_COLUMN_HEADER": "Addon",
         "ADDON_INSTALL_BUTTON": "Installieren",
         "ADDON_UPDATE_BUTTON": "Aktualisieren",
-        "AUTHOR_COLUMN_HEADER": "Autor",
-        "AUTO_UPDATE_ICON_TOOLTIP": "Auto-Update aktiviert",
+        "AUTHOR_COLUMN_HEADER": "Veröffentlicher",
+        "AUTO_UPDATE_ICON_TOOLTIP": "Automatisches Update aktiviert",
         "GAME_VERSION_COLUMN_HEADER": "Spielversion",
         "LATEST_VERSION_COLUMN_HEADER": "Aktuelle Version",
         "PROVIDER_COLUMN_HEADER": "Anbieter",
         "PROVIDER_RELEASE_CHANNEL": "Anbieter Kanal",
-        "RELEASED_AT_COLUMN_HEADER": "Erscheinungsdatum",
+        "RELEASED_AT_COLUMN_HEADER": "Veröffentlichung",
```

Comment:
better is "Aktualisierungsdatum" or "Aktualisiert am/vor"
Veröffentlichung is wrong, that is the date of the very first release of that add-on, not the update date